Tasks/AzureWebAppContainerV1/task.json

```diff
@@ -95,12 +95,20 @@
         },
         {
             "name": "imageName",
```

Comment:
Please syncup with @azooinmyluggage  or @RoopeshNair  for updating the help strings.

Reply:
@azooinmyluggage , Can you please provide help strings for this too? The help text should convey to provide the image name for single container and "replaceable images" for multi-containers.
Should we add an alias as "images" for multi-container?